app/parse.py

```diff
@@ -8,5 +13,18 @@
     duration: str
 
 
+def parse_single_course(course_soup: Tag) -> Course:
+    name = course_soup.select_one("a.ProfessionCard_title__Zq5ZY > h3").text
+    short_description = course_soup.select_one(
+        ".typography_landingTextMain__Rc8BD.mb-32").text
+    duration = course_soup.select("span")[-2].text
```

Comment:
The extraction of `duration` using `select("span")[-2]` assumes that the duration is always the second-to-last `span` element. This approach is fragile and may fail if the structure of the webpage changes. Consider using a more reliable method to locate the duration.